### PR TITLE
Remove the unnecessary Netty dependency so we can trim our plugin size

### PR DESCRIPTION
**Pull request by kiiadi**
Saves 3m on the packaged plugin.zip

### CODE REVIEW

**Comment by abrooksv** [comment]
Do we need software.amazon.awssdk:netty-nio-client?

**Comment by kiiadi** [comment]
> Do we need software.amazon.awssdk:netty-nio-client?

Nope - changed - and that gets rid of all the reactive streams netty dependencies too - doesn't really affect the size tho - still 19MB.

File: build.gradle [hunk 1]
```diff
@@ -68,6 +68,7 @@ allprojects {
         runtimeClasspath.exclude group: "org.slf4j"
         runtimeClasspath.exclude group: "org.jetbrains.kotlin"
         runtimeClasspath.exclude group: "org.jetbrains.kotlinx"
+        runtimeClasspath.exclude group: "software.amazon.awssdk", module: "netty-nio-client"
```
Comment (hunterwerlla) [inline build.gradle:71]: Did you accidentally remove the change?

Comment (kiiadi) [reply build.gradle:71]: No - there's no need to explicitly exclude the netty dependency - by removing the `nett-nio-client` - its dependencies don't get resolved.

**Pull request merged**